contracts/v2/periphery/BatchL2DataCreatedRollup.sol

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.20;
+
+import "../lib/PolygonRollupBaseEtrog.sol";
+
+contract BatchL2DataCreatedRollup {
```

Comment:
Add a comment that this contract is used for a specific tool. Add the link of the tool.
Add the link of the code that this contract has been copied from for more context